client/src/pages/feedback.tsx

```diff
@@ -190,7 +257,14 @@ function FeedbackItem(props: {
                 flexGrow: 1,
               }}
               renderOption={(option) => (
-                <Typography align="left">
+                <Typography
+                  style={{
+                    color:
+                      option.status == Status.INCOMPLETE ? "grey" : "black",
```

Comment:
use === instead of ==
This explains the difference: https://howtodoinjava.com/typescript/equals-vs-strict-equals/